CHANGELOG.md

```diff
@@ -15,6 +16,7 @@ script to deploy datasource within namespace/tenant-level instances of Grafana w
   * [UPGRADE] Fluent Bit Helm chart upgraded from 0.47.10 to 0.48.6
   * [UPGRADE] Elasticsearch Exporer Helm chart upgraded from 6.5.0 to 6.6.1
   * [UPGRADE] OpenSearch Data Source Plugin to Grafana upgraded from 2.21.1 to 2.23.1
+>>>>>>> main
```

Comment:
Looks like this was left over from a merge conflict fix.

Reply:
oh...good catch.  Glad it was only in the CHANGELOG.md.  Thanks.